addons/mail/static/tests/mock_server/mock_models/res_partner.js

```diff
@@ -170,6 +172,14 @@ export class ResPartner extends webModels.ResPartner {
                 makeKwArgs({ fields: { channel: [], persona: [] } })
             );
         }
+        const internalUsers = ResUsers._filter([]).filter((user) => !user.partner_share);
+        for (const user of internalUsers) {
+            store.add(this.browse(user.partner_id), {
+                groups_id: user.groups_id.includes(channel.group_public_id)
+                    ? channel.group_public_id
+                    : undefined,
+            });
+        }
```

Comment:
Should adapt mock server to match python implementation, even if this means diffing in whole method to match the `.search()` and extra_domain, the intent is to copy python code to make mock server look as close to python as we can